### DISCUSSION TITLE
how to help debug when alerts missing from "area" coverage

### DISCUSSION THREAD

**Discussion by JJKraw** [weather-gov/api#325] [Q&A]
I've seen this a couple of times and could use some guidance in what data to gather to help figure out why.

The issue is a discrepancy between alerts reported via API endpoint alerts/active/area/(state code) and alerts/active/zone/(forecast zone code). In my case the state is MA and the zone is MAZ014.

There is a special weather statement currently active across almost the entire BOX area. That one is OK. But we've had a couple of short duration SWSs for thunderstorms (the type that include a polygon for the applicable area). The first of these initially showed up both under area/MA and zone/MAZ014 but shortly thereafter disappeared from area/MA (while remaining in zones/MAZ014). It eventually came back to area/MA after another SWS and then a severe thunderstorm warning were issued.

Given the above situation I need to pay attention to the weather now (spotter and all that) but when someone gets a chance let me know how to help debug. Of if I'm explained this badly in my haste, I'll try to clarify what's happening. :)

**Comment by scadergit** [comment]
I believe this is associated to a known issue where a stale response is being returned by our CDN. We have requested to deploy a configuration change to address this, but there is a not a specific date yet.

**Comment by JJKraw** [comment]
Thanks. I'm here if you need any data.
